front-end/src/components/Content/Post/Post.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./style.scss";
+
+export default function Post() {
+  const item = (
```

Comment:
@Taiga17 -san
The code is correct but I have a new solution better than before. You can refer it.

```JS
function renderItems() {
    return Array.from(new Array(12)).map((item, index) => (
      <div class="item" key={index}>
          <img src={"./images/card-image.png"} class="card-image" />
          <h3 className="title">Card Title</h3>
          <p className="card-text">
            This is a longer card with supporting text below as a natural lead-in to
            additional content.
          </p>
      </div>
    ));
  }
  
  return <div class="items">{renderItems()}</div>;
```